.github/workflows/push-image-ghcr.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Docker Image to GHCR
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump-version:
+        type: choice
+        required: true
+        description: Bump version
+        default: patch
+        options:
+          - patch
+          - minor
+          - major
+          - none
+
+jobs:
+  bump-version:
+    name: "Bump Image VERSION"
+    if: inputs.bump-version != 'none'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Auto Bump
+        run: ./scripts/bump-version.sh ${{inputs.bump-version}}
+      - uses: stefanzweifel/git-auto-commit-action@v5
+  push-store-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout GitHub Action"
+        uses: actions/checkout@main
+      - name: "Login to GitHub Container Registry"
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: "Build & Push Terminology Server Image"
```

Comment:
I would recommend we build multi-platform images, as supporting X86 and arm is almost mandatory in 2024 IMHO.
You can take a look at what we do for the Medplum Devbox for an exemple: https://github.com/bonfhir/medplum-devbox/blob/80a7fac277fa149d57749703a5b5473ac87dc018/.github/workflows/build_docker.yml#L34
Also - https://docs.docker.com/build/ci/github-actions/multi-platform/